### PR TITLE
#1718 Make log layout customizable

### PR DESCRIPTION
**Pull request by gerashegalov**
Users should be able to use a custom job log format such as GMT timestamps

### CODE REVIEW

**Comment by gerashegalov** [comment]
@kunkun-tang thanks for reviewing, added tests

**Comment by gerashegalov** [comment]
Closing/reopening PR to trigger travis

**Comment by gerashegalov** [comment]
Hi @kunkun-tang, thanks for checking, test code added!

File: azkaban-common/src/main/java/azkaban/flow/CommonJobProperties.java [hunk 1]
```diff
@@ -164,4 +164,8 @@
   public static final String FLOW_START_TIMEZONE =
       "azkaban.flow.start.timezone";
 
+  /**
+   * Format string for Log4j's EnhancedPatternLayout
+   */
+  public static final String JOB_LOG_LAYOUT = "azkaban.job.log.layout";
```
[the remark this answers is not in the document]
Comment (gerashegalov) [reply azkaban-common/src/main/java/azkaban/flow/CommonJobProperties.java]: 👍

[the remark this answers is not in the document]
Comment (gerashegalov) [reply azkaban-common/src/main/java/azkaban/flow/CommonJobProperties.java]: build passed finally

**Comment by HappyRay** [comment]
What Travis test failures did you run into? We would like to know so that we can fix the flaky tests.

**Comment by gerashegalov** [comment]
Thanks for reviewing and committing this PR, @kunkun-tang ! 

@HappyRay one failure was https://api.travis-ci.org/v3/job/363278283/log.txt
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':azkaban-web-server:compileTestJava'.
> java.lang.reflect.InvocationTargetException
```

**Pull request merged**